### DISCUSSION TITLE
Connect to mysql via socket

### DISCUSSION THREAD

**Discussion by milindpatel63** [Kovah/LinkAce#407] [Support] [answered]
I am running linkace in docker and mariadb on the same machine.
I would like to connect to MySQL via socket....
Is there any way to do so right now?

**Comment by milindpatel63** [comment, marked as answer]
NVM...found in the php file...
it was DB_SOCKET env variable

**Comment by Kovah** [comment]
Running LinkAce with a socket connection to MySQL is possible by using the `DB_SOCKET` option in your .env file.

Like: `DB_SOCKET= /var/run/mysqld4.sock`
Then remove the `DB_HOST` option.

That should work.
